turcica.csl

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" demote-non-dropping-particle="sort-only" xmlns="http://purl.org/net/xbiblio/csl">  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>turcica</title>
+    <id>http://www.zotero.org/styles/turcica</id>
+    <link rel="self" href="http://www.zotero.org/styles/turcica"/>
+	<link href="https://poj.peeters-leuven.be/content.php?journal_code=TURC&url=journal" rel="documentation"/>
```

Comment:
It's not a fan of ampersans.
Please replace with `<link href="https://poj.peeters-leuven.be/content.php?journal_code=TURC&amp;url=journal" rel="documentation"/>`